crates/bitcoin-da/src/spec/header.rs

```diff
@@ -71,7 +70,7 @@ impl HeaderWrapper {
 }
 
 /// BitcoinHeaderWrapper is a wrapper around BitcoinHeaderWrapper to implement borsh serde
-#[derive(Clone, PartialEq, Eq, Debug, Hash, Deserialize, Serialize)]
+#[derive(Clone, PartialEq, Eq, Debug, Hash, Serialize, Deserialize)]
 #[repr(transparent)]
 #[serde(transparent)]
 pub struct BitcoinHeaderWrapper {
```

Comment:
If it's just a wrapper, imo it's better to have it as `pub struct BitcoinHeaderWrapper(BitcoinHeader)`